export_helpers/containerize_readme.md

```diff
@@ -20,6 +20,13 @@ Install on openSUSE 13.2:
 
 Log out and in again to refresh the user's group.
 
+## Set Up
+
+When neccesarry, we've included a `setup.rb` script in your new containerized
+application. Please run it before starting docker-compose.
+
+    ./setup.rb
```

Comment:
We could go one step further and wrap running setup and docker-compose in one `start` script, so that it's really just one step for the user to get the containers up and running. But I'm ok with the solution for now. We can refine it later, if needed.